[2023-10-30T11:20:27.310+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: random_names.streaming_to_kafka manual__2023-10-30T11:20:24.774146+00:00 [queued]>
[2023-10-30T11:20:27.317+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: random_names.streaming_to_kafka manual__2023-10-30T11:20:24.774146+00:00 [queued]>
[2023-10-30T11:20:27.317+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-10-30T11:20:27.332+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): streaming_to_kafka> on 2023-10-30 11:20:24.774146+00:00
[2023-10-30T11:20:27.336+0000] {standard_task_runner.py:57} INFO - Started process 236 to run task
[2023-10-30T11:20:27.339+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'random_names', 'streaming_to_kafka', 'manual__2023-10-30T11:20:24.774146+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmplcxna11h']
[2023-10-30T11:20:27.343+0000] {standard_task_runner.py:85} INFO - Job 45: Subtask streaming_to_kafka
[2023-10-30T11:20:27.401+0000] {task_command.py:415} INFO - Running <TaskInstance: random_names.streaming_to_kafka manual__2023-10-30T11:20:24.774146+00:00 [running]> on host df75f57c0534
[2023-10-30T11:20:27.487+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='random_names' AIRFLOW_CTX_TASK_ID='streaming_to_kafka' AIRFLOW_CTX_EXECUTION_DATE='2023-10-30T11:20:24.774146+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-30T11:20:24.774146+00:00'
[2023-10-30T11:20:29.250+0000] {logging_mixin.py:151} INFO - {'full name': 'Mr Daryl Dean', 'gender': 'male', 'location': 'Westheimer Rd 6521', 'city': 'Brisbane', 'country': 'Australia', 'postcode': 1874, 'latitude': '-88.1521', 'longtitude': '-165.9317', 'email': 'daryl.dean@example.com'}
[2023-10-30T11:20:29.256+0000] {conn.py:380} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:29092 <connecting> [IPv4 ('172.22.0.5', 29092)]>: connecting to kafka:29092 [('172.22.0.5', 29092) IPv4]
[2023-10-30T11:20:29.257+0000] {conn.py:1205} INFO - Probing node bootstrap-0 broker version
[2023-10-30T11:20:29.258+0000] {conn.py:410} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:29092 <connecting> [IPv4 ('172.22.0.5', 29092)]>: Connection complete.
[2023-10-30T11:20:29.365+0000] {conn.py:1267} INFO - Broker version identified as 2.5.0
[2023-10-30T11:20:29.366+0000] {conn.py:1268} INFO - Set configuration api_version=(2, 5, 0) to skip auto check_version requests on startup
[2023-10-30T11:20:29.372+0000] {logging_mixin.py:151} INFO - KafkaProducer connected successfully.
[2023-10-30T11:20:29.529+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:20:29.530+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:20:29.530+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:20:29.531+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:29.633+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:20:29.633+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:20:29.634+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:20:29.634+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:29.736+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:20:29.737+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:20:29.737+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:20:29.738+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:29.840+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:20:29.841+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:20:29.842+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:20:29.842+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:30.046+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:20:30.047+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:20:30.048+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:20:30.048+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:30.252+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:20:30.252+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:20:30.253+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:20:30.253+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:30.507+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:20:30.509+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:20:30.509+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:20:30.510+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:30.763+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:20:30.764+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:20:30.765+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:20:30.765+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:31.220+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:20:31.222+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:20:31.225+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:20:31.226+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:31.682+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:20:31.683+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:20:31.684+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:20:31.684+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:32.491+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:20:32.492+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:20:32.493+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:20:32.494+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:33.302+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:20:33.302+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:20:33.302+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:20:33.303+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:34.314+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:20:34.314+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:20:34.314+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:20:34.315+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:35.326+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:20:35.326+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:20:35.327+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:20:35.327+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:36.438+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:20:36.439+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:20:36.439+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:20:36.440+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:37.552+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:20:37.553+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:20:37.553+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:20:37.554+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:38.514+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:20:38.514+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:20:38.514+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:20:38.515+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:39.472+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:20:39.474+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:20:39.475+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:20:39.476+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:40.684+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:20:40.685+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:20:40.685+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:20:40.685+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:41.894+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:20:41.894+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:20:41.895+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:20:41.895+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:43.002+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:20:43.003+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:20:43.003+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:20:43.003+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:44.115+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:20:44.116+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:20:44.117+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:20:44.118+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:45.330+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:20:45.332+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:20:45.332+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:20:45.333+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:46.545+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:20:46.546+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:20:46.547+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:20:46.547+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:47.761+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:20:47.762+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:20:47.763+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:20:47.763+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:48.976+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:20:48.978+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:20:48.978+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:20:48.979+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:49.839+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:20:49.840+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:20:49.840+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:20:49.841+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:50.700+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:20:50.701+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:20:50.701+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:20:50.702+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:51.713+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:20:51.714+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:20:51.714+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:20:51.715+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:52.726+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:20:52.727+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:20:52.728+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:20:52.729+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:53.791+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:20:53.792+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:20:53.793+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:20:53.793+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:54.854+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:20:54.856+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:20:54.856+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:20:54.857+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:55.969+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:20:55.970+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:20:55.970+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:20:55.970+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:57.082+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:20:57.082+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:20:57.083+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:20:57.084+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:58.044+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:20:58.046+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:20:58.046+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:20:58.047+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:20:59.007+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:20:59.008+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:20:59.008+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:20:59.009+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:00.273+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:21:00.274+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:21:00.275+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:21:00.276+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:01.538+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:21:01.539+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:21:01.540+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:21:01.540+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:02.703+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:21:02.703+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:21:02.703+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:21:02.704+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:03.865+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:21:03.866+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:21:03.866+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:21:03.867+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:05.080+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:21:05.081+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:21:05.082+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:21:05.083+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:06.296+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:21:06.297+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:21:06.297+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:21:06.298+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:07.561+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:21:07.562+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:21:07.563+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:21:07.564+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:08.827+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:21:08.828+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:21:08.829+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:21:08.829+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:09.790+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:21:09.790+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:21:09.790+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:21:09.790+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:10.751+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:21:10.751+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:21:10.752+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:21:10.753+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:11.814+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:21:11.816+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:21:11.816+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:21:11.817+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:12.878+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:21:12.878+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:21:12.878+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:21:12.879+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:14.137+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:21:14.138+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:21:14.138+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:21:14.138+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:15.396+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:21:15.397+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:21:15.397+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:21:15.398+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:16.406+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:21:16.407+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:21:16.408+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:21:16.409+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:17.420+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:21:17.420+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:21:17.421+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:21:17.422+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:18.332+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:21:18.333+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:21:18.334+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:21:18.334+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:19.244+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:21:19.245+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:21:19.246+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:21:19.246+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:20.157+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:21:20.158+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:21:20.158+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:21:20.159+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:21.069+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:21:21.069+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:21:21.070+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:21:21.070+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:21.980+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:21:21.980+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:21:21.981+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:21:21.981+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:22.889+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:21:22.890+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:21:22.891+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:21:22.892+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:23.903+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:21:23.904+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:21:23.904+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:21:23.904+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:24.915+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:21:24.916+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:21:24.916+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:21:24.917+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:25.927+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:21:25.928+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:21:25.928+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:21:25.928+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:26.938+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:21:26.938+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:21:26.939+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:21:26.940+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:28.102+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: connecting to localhost:29092 [('127.0.0.1', 29092) IPv4]
[2023-10-30T11:21:28.103+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]> returned error 111. Disconnecting.
[2023-10-30T11:21:28.103+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv4 ('127.0.0.1', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2023-10-30T11:21:28.104+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:29.265+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: connecting to localhost:29092 [('::1', 29092, 0, 0) IPv6]
[2023-10-30T11:21:29.266+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]> returned error 99. Disconnecting.
[2023-10-30T11:21:29.266+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=localhost:29092 <connecting> [IPv6 ('::1', 29092, 0, 0)]>: Closing connection. KafkaConnectionError: 99 EADDRNOTAVAIL
[2023-10-30T11:21:29.267+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2023-10-30T11:21:29.374+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/scripts/script.py", line 52, in start_streaming
    future = producer.send('random_names', value=data)
  File "/home/airflow/.local/lib/python3.8/site-packages/kafka/producer/kafka.py", line 576, in send
    self._wait_on_metadata(topic, self.config['max_block_ms'] / 1000.0)
  File "/home/airflow/.local/lib/python3.8/site-packages/kafka/producer/kafka.py", line 702, in _wait_on_metadata
    raise Errors.KafkaTimeoutError(
kafka.errors.KafkaTimeoutError: KafkaTimeoutError: Failed to update metadata after 60.0 secs.
[2023-10-30T11:21:29.463+0000] {taskinstance.py:1398} INFO - Marking task as FAILED. dag_id=random_names, task_id=streaming_to_kafka, execution_date=20231030T112024, start_date=20231030T112027, end_date=20231030T112129
[2023-10-30T11:21:29.520+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 45 for task streaming_to_kafka (KafkaTimeoutError: Failed to update metadata after 60.0 secs.; 236)
[2023-10-30T11:21:29.571+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-30T11:21:29.619+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
